crypto/rand/rand_lib.c

```diff
@@ -286,7 +288,14 @@ static uint64_t get_timer_bits(void)
             return TWO32TO64(tv.tv_sec, tv.tv_usec);
     }
 # endif
-    return time(NULL);
+    {
+        time_t t;
+
+        t = time(NULL);
```

Comment:
nit: `t`  could be initialized immediately in the definition `time_t t = time(NULL);`